app/assets/stylesheets/layout.css.scss

```diff
@@ -92,14 +98,25 @@ a {
   margin-top: 10px;
 }
 
-#cart {
-  float: right;
-  margin-top: 10px;
-  padding-right: 40px;
+.container {
+  width: 100%;
+  padding-right: 0px;
+  padding-left: 0px;
+  background-color: white;
+  // margin-top: 170px;
```

Comment:
lots of commented-out css that needs to be removed.  This is the last time I am going to warn you before firing you from the project.